src/tasks-api/BusinessLayer/ILearningTaskService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using EKids.Chatbot.Tasks.DataAccessLayer.Entities;
+
+namespace EKids.Chatbot.Tasks.BusinessLayer;
+public interface ILearningTaskService
+{
+    Task AddTask(LearningTask task, Guid courseId, CancellationToken cancellation);
```

Comment:
Add Summary please (And in other Interfaces too)

Reply:
Added summaries